### ISSUE TITLE
Doesn't work reliable with coffeescript, watch and Gulp

### ISSUE THREAD

**Issue by heikkimu** [yuanqing/gulp-tape#16]
When tests are watching file changes, it won't take fresh files until the gulp has been restarted.

Reproduce:

``` coffeescript
//index.coffee
hello = () ->
  return 1 //Try to change this value to see the changes in tests-results.

module.exports.hello = hello

```

``` coffeescript
//./test/test.coffee
test = require 'tape'
index = require '../index'

test 'if the test fails when you will manipulate index.coffee', (t) ->
  t.equal index.hello(), 1
  t.end()
```

``` javascript
// npm install gulp gulp-cjsx gulp-tape tape --save-dev
// to start, type: gulp
var gulp = require('gulp');
var coffee = require('gulp-coffee');
var tape = require('gulp-tape');

gulp.task('coffee', function() {
  gulp.src('./**/*.coffee')
    .pipe(coffee({bare: true})
    .pipe(gulp.dest('./dist/'));
});

gulp.task('test', function() {
  gulp.src('./dist/test/*.js')
    .pipe(tape());
});

gulp.task('default', [], function() {
    gulp.watch(['./**/*.coffee'], ['coffee', 'test']);
});
```

**Comment by kjellmorten** [comment]
As a temporary workaround, I use a gulp task to flush the node require cache, and run it in the `watch` task, before the tests.

```
gulp.task('flush', function () {
  for (var prop in require.cache) {
    if (prop.indexOf("node_modules") === -1) {
      delete require.cache[prop];
    }
  }
});
```

(Code borrowed from https://github.com/sindresorhus/gulp-jasmine/issues/32)

Hoping for a solution in gulp-tape, though.

**Comment by sdbondi** [comment]
Same issue here.

`gulp-tape` is already using require-uncached to bypass the cache for 
files passed on via `gulp.src`. 

https://github.com/yuanqing/gulp-tape/blob/v0.0.9/index.js#L36

Despite this, I've had to use `requireUncached` to require source 
and test files so that they refresh. Clearing the require cache as @kjellmorten 
says will also work.

Looks like `gulp-tape` should refresh the require cache for gulp.src'd files.
So not sure why this happens.

**Comment by yuanqing** [comment]
Closing because this issue should be fixed as of `0.0.10`
